# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/darktable/hotfix-libjpeg.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- darktable-5.0.0/src/external/rawspeed/src/librawspeed/decompressors/JpegDecompressor.cpp.vanilla	2025-01-03 16:10:29.399875775 +0100
+++ darktable-5.0.0/src/external/rawspeed/src/librawspeed/decompressors/JpegDecompressor.cpp	2025-01-03 16:10:47.556542791 +0100
@@ -51,7 +51,7 @@
 }
 
 boolean fill_input_buffer(j_decompress_ptr cinfo) {
-  return cinfo->src->bytes_in_buffer != 0;
+  return (boolean)(cinfo->src->bytes_in_buffer != 0);
 }
 
 // NOLINTNEXTLINE(google-runtime-int)
